group_vars/location_rixbox/networks.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+ipv6_prefix: '2001:bf7:820:800::/56'
+
+networks:
+  - vid: 20
+    role: mesh
+    name: mesh_core
+    prefix: 10.230.3.164/32
+    ipv6_subprefix: -1
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.230.39.0/24
+    ipv6_subprefix: 0
+    assignments:
+      rixbox-core: 1
+    # inbound_filtering: true
+    # enforce_client_isolation: true
+
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.154.112/28
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      rixbox-core: 1
+      rixbox-ap1: 2
+      rixbox-ap2: 3
+      rixbox-ap3: 4
+      # rixbox-n-5ghz: 5 # AirOS LiteBeam RHNK
```

Comment:
Stations are usually called $local_location-$remote_location. While this is no written rule i like to encourage you to change it in order to follow the current pattern.

in this case it would be rixbox-rhnk.

Please also dont comment out "unmanaged" devices, the definition is stil useful for name resolution.